City-of-Helsinki/drupal-helfi-kasvatus-koulutus
# Use contextual filter to get taxonomy term name at daycare search

# Use contextual filter to get taxonomy term name
At the daycare search, use contextual filter to get the taxonomy term name. This removes the need to directly use the taxonomy content, which causes problems when importing configs to different environments.

Also fixes a problem when importing config: "Unexpected error during import (–––): The readonly_field_widget plugin does not exist".

## How to install
* Get the changes from this branch.
* `composer require drupal/helfi_tpr:dev-UHF-X-ontology-word-display-view-fix`
* Make sure your Kasko instance is up and running.
* Run `make drush-cim drush-cr`
* **Also** test separately with `make fresh`.
* **Also** test separately with `make new`.

## How to test
* Check that there are no problems when importing the configs.
* Go to a page containing the daycare search.
* Check that the result list includes results that has the "Daycare" term set as unit type, and nothing else.

## Other PRs
* https://github.com/City-of-Helsinki/drupal-module-helfi-tpr/pull/114
